library/core/src/mem/mod.rs

```diff
@@ -580,6 +580,21 @@ pub const fn needs_drop<T>() -> bool {
     intrinsics::needs_drop::<T>()
 }
 
+/// Returns `true` if finalizing values of type `T` inside GC matters.
```

Comment:
I'm struggling to parse this sentence: can we maybe spell out what "matters" means? I guess it means "Returns false if values of type `T` don't need finalising or true otherwise". [I think this might be one of those rare-ish cases where expressing things in terms of the negative case might be easier!]

Reply:
> "Returns false if values of type T don't need finalising or true otherwise"

I'm happy with this. Do you think it's obvious that when talking about finalizers it's always within the context of `Gc<T>` so we don't need to explicitly mention this?

Reply:
Fixed: 06ee0e6f8d034786d5145c1f35c49a06e5e3a903

Reply:
> Do you think it's obvious that when talking about finalizers it's always within the context of Gc<T> so we don't need to explicitly mention this?

Good question. It rarely hurts to be clearer about these things, especially as this is a "general" part of the compiler. Maybe we should add the "GC" bit back then?

Reply:
Perfect!